cpp/ql/test/library-tests/dataflow/taint-tests/map.cpp

```diff
@@ -68,7 +68,7 @@ void test_pair()
  	i.swap(j);
 	k.swap(l);
 	sink(i.first);
-	sink(i.second); // $ MISSING: ast,ir
+	sink(i.second); // $ ir, MISSING: ast
```

Comment:
This shows that the change is doing what we want for the IR based taint tracking.